### PR TITLE
arch_updates: new module from scratch

### PR DESCRIPTION
**Pull request by lasers**
Hi, I made `arch_updates`.

Instead of fixing up anything, I made it from scratch. This is similar to `void_linux` where we can get package names, versions, etc... and we can print individual update numbers too. Very nice if used with color eg `UPD (blue) {pacman}, (orange) {foreign)`. This is optimized too.

Requires: `checkupdates` (owned by pacman), `cower` (optional), and `Pull Request` https://github.com/ultrabug/py3status/pull/971

Default.
0-count
Update shield.
0-shield
Individual counts.
0-separate
Colors. I show colors now. Blue + orange = green.
divider
Blue `PAC`, orange `FOR`, or green `UPD`.
1-color-based
Individual counts.
1-separate
Blue, orange, or mixed `UPDATE`.
1-mixed
Blue, orange, or green `UPDATE`.
1-seperate-color
Name of updates. 
1-updates

You get the idea. If we updates regularly, it's not a problem. Sometimes we will just get a sudden large influx of updates as shown in this picture... or I'm just lazy with updates. Either way, I added `max_length=999` in the examples too for the users.

EDIT: Name of updates after an update! :-)
1-updates-new

Pending:
* We can't use `if=count&color=update UPDATE`. Requires `{count}`, `*foreign}`, or `*pacman}`
* We can't use `if=foreign&color=foreign FOREIGN`. Requires `*foreign}`.
* We can't use `if=pacman&color=pacman PACMAN`. Requires `*pacman}`.

### CODE REVIEW

**Comment by lasers** [comment]
This is ready. Deprecation included.

**Comment by lasers** [comment]
@maximbaz You should try this. I think using `show update bars` is fun as it does not give me numbers. However, I think I'm switching bars with blocks `\u25b0`.

File: py3status/modules/arch_updates.py [hunk 1]
```diff
@@ -1,127 +1,300 @@
...
+    format += '[\?max_length=999 {format_pacman}][\?soft  ]'
+    format += '[\?max_length=999 {format_foreign}]]'
+    format_pacman = '\?color=pacman {name}'
+    format_foreign = '\?color=foreign {name}'
+    format_pacman_separator = ' '
+    format_foreign_separator = ' '
+}
+
+# show update bars
+apt_updates {
```
Comment (maximbaz) [inline py3status/modules/arch_updates.py]: arch_updates

**Comment by maximbaz** [comment]
This is very cool indeed, especially with blocks!

File: py3status/modules/arch_updates.py (hunk 1 above)
Comment (lasers) [reply py3status/modules/arch_updates.py]: Thanks. Fixed. 👍

**Comment by lasers** [comment]
>@ultrabug closing, specialised updates modules is not our way forward

@ultrabug @tobes Do you want to close this too? And #1116?

**Comment by tobes** [comment]
yeah I'm going to close this too.

**Pull request closed without merging**